recipes/misc/wtmptrim.oe

```diff
@@ -0,0 +1,19 @@
+SUMMARY = "Utility for keeping the tail of wtmp/btmp files"
+LICENSE = "GPL-2.0+"
+SRC_URI = "file://wtmptrim.c"
+COMPATIBLE_HOST_ARCHS = ".*linux"
+
+inherit c
+
+S = "${SRCDIR}"
+do_compile() {
+    ${CC} -Wall -Wextra -Werror -O2 -g -o wtmptrim wtmptrim.c
```

Comment:
As in, set CFLAGS to `-Wall -Wextra -Werror -O2 -g` and substitute that, or append `Wall -Wextra -Werror` to the existing CFLAGS, or just use the existing CFLAGS without adding `-W*`?